distro/src/avail/Avail.avail/Foundation.avail/Late Math.avail

```diff
@@ -650,7 +641,8 @@ Public method "_↑÷=_" is
 	divisor : number
 |
 	var ?= eject var ÷ divisor;
-] : ⊤;
+] : ⊤
+styled by statement styler;
```

Comment:
Not convinced by this at all. These concepts would be statements in most languages, and _are_ in fact statements in Avail. And since they behave like first-order statements, we might as well style them as statements.